app/assets/javascripts/app/views/conversations_inbox_view.js

```diff
@@ -0,0 +1,87 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
+
+app.views.ConversationsInbox = Backbone.View.extend({
+  el: "#conversations-container",
+
+  events: {
+    "click .conversation-wrapper": "displayConversation",
+    "click .new-conversation-btn": "displayNewConversation"
+  },
+
+  initialize: function() {
+    new app.views.ConversationsForm({contacts: gon.contacts});
+    this.setupConversation();
+  },
+
+  renderConversation: function(conversationId) {
+    if (parseInt("" + conversationId, 10)) { // Ensures that conversationId is a readable number
```

Comment:
I think we should check this in the backbone router instead before calling this function.